bazel/cargo/wasmtime/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 edition = "2021"
 name = "wasmtime-c-api-bazel"
-version = "9.0.3"
-rust-version = "1.66.0"
+version = "24.0.0"
+rust-version = "1.78.0"
```

Comment:
AFAIR the last update of `rules_rust` pinned the Rust compiler to v1.77.2, so it seems that Bazel doesn't enforce this check (just an observation, nothing actionable).

Reply:
Yeah that's what I figured as wells maybe it's just the initial/default version?

Reply:
I don't think so, you can look verify the Rust version used inside Bazel using:
```
./bazel-proxy-wasm-cpp-host/external/rust_linux_aarch64__wasm32-wasi__stable_tools/bin/rustc --version
rustc 1.77.2 (25ef9e3d8 2024-04-09
```